PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -1376,10 +1376,11 @@ void trim(generic_string& str)
 bool endsWith(const generic_string& s, const generic_string& suffix)
 {
 #if defined(_MSVC_LANG) && (_MSVC_LANG > 201402L)
-#error Replace this function with basic_string::ends_with
-#endif
+	return s.ends_with(suffix);

```

Comment:
If C++20 is enabled globally, we can leave just the `ends_with` call and remove #ifdef and #else.
https://learn.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=msvc-170

Reply:
@mere-human 
Could you explain why these following lines please?
```
#if defined(_MSVC_LANG) && (_MSVC_LANG > 201402L)
#error Replace this function with basic_string::ends_with
#endif
```

Reply:
@mere-human 
Good point.
I will do it after merging this PR.

Reply:
Finally, it should be kept due to the CI (Appveyor) doesn't provide GCC 9 package.
It's kind of stupid that C++20 cannot be used due to the auxiliary compiler.
@chcg Do you have any thought about it?

Reply:
@donho  how about enable C++20 in gcc makefile too?

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/3b8446d8cec31b02f752e644e7d8634d9042d0f4/PowerEditor/gcc/makefile#L39-L41